v1-2/learn/api-docs/ballerina/observe/index.html

```diff
@@ -603,7 +603,7 @@ <h2>Module Overview</h2>
 Ballerina supports Observability out of the box. This module provides user api's to make Ballerina Observability more flexible for the user.</p>
 <p>To observe Ballerina code, the '--b7a.observability.enabled=true' property should be given when starting the service.
 i.e. `ballerina run hello_world.bal --b7a.observability.enabled=true'
-For more information on Ballerina Observability visit <a href="https://v1-0.ballerina.io/learn/how-to-observe-ballerina-code/">How to Observe Ballerina Services</a>.</p>
+For more information on Ballerina Observability visit <a href="https://v1-0.ballerina.io/v1-2/learn/how-to-observe-ballerina-code/">How to Observe Ballerina Services</a>.</p>
```

Comment:
`https://v1-0.ballerina.io/v1-2/learn/how-to-observe-ballerina-code/` is wrong

Reply:
Fixed.